### PR TITLE
EIP-4400: Mark for review; Update motivation

### PR DESCRIPTION
**Pull request by Daniel-K-Ivanov**
Marking this EIP for review. Since the inception of this EIP, there have been 2 implementations and there was no feedback on changing the spec.

### CODE REVIEW

File: EIPS/eip-4400.md [hunk 1]
```diff
@@ -183,6 +187,11 @@ contract ERC721Consumable is IERC721Consumable, ERC721 {
 }
 ```
 
+Other implementations of the `Consumable` role:
```
Comment (lightclient) [inline EIPS/eip-4400.md:190]: Please remove this note to other implementations. EIPs are not an appropriate place to advertise projects.

**Review by lightclient — CHANGES_REQUESTED** [review]

**Comment by 5534vovan** [comment]
> Marking this EIP for review. Since the inception of this EIP, there have been 2 implementations and there was no feedback on changing the spec.

**Comment by Daniel-K-Ivanov** [comment]
@lightclient @axic friendly reminder for reviewing the PR 🙏

**Review by 5534vovan — APPROVED** [review]

**Review by Bastek8 — APPROVED** [review]

**Review by lightclient — APPROVED** [review]
LGTM

**Comment by Andrel2s** [comment]
Go

**Pull request merged**